tests/integration/test_logcollector/test_macos/test_macos_log_process.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import platform
+import signal
+
+import wazuh_testing.logcollector as logcollector
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.services import search_process, control_service
+from wazuh_testing.tools.utils import retry
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
```

Comment:
To follow the new development guides, we must implement the common fixture `file_monitoring`